src/handler.ts

```diff
@@ -27,14 +27,21 @@ export async function requestAPI<T>(
     throw new ServerConnection.NetworkError(error as any)
   }
 
-  let data: any = await response.text()
+  // let data: any = await response.text()
```

Comment:
Another instance here. If you agree that we should generally add a comment above commented out code in the repo, can you add a comment above this explaining under what scenario we should uncomment it?

Reply:
This commented code should have been deleted, it just wasn't on accident.